.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Bazel build
+
+on:
+    push: {}
+    pull_request: {}
+
+jobs:
+    build:
+        name: Build tcmalloc
+        runs-on: ubuntu-latest
+        steps:
+
+        - name: Checkout repository
+          uses: actions/checkout@v2
+
+        - name: Setup bazel
+          uses: jwlawson/actions-setup-bazel@v1
```

Comment:
Is this step necessary?

It looks like some of the virtual environments for Github Actions already install bazel (or have a helper ready to go): https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/installers/bazel.sh

I'm also looking at https://github.com/googleapis/google-cloud-cpp/pull/4333/files for reference.